Not working with `npm i -D cli-typescript-class-organizer`

Hi thanks for your contribution 👍
   
   I installed it with `npm i -D cli-typescript-class-organizer` for own strategy. 
   Then, run it from package.json with `"tsco" : "tsco --help"` but seem to be not working and throw this error. 
   Please help.

```
Error: Cannot find module '/Users/data/projects/click-ui/index.ts'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at Object.<anonymous> (/Users/data/projects/click-ui/node_modules/ts-node/src/bin.ts:157:12)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
    at Module.load (internal/modules/cjs/loader.js:977:32)
    at Function.Module._load (internal/modules/cjs/loader.js:877:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
```

--- a/bin.js
+++ b/bin.js
@@ -1,0 +1,8 @@
+#!/usr/bin/env node
+
+// to enable --files flag from ts-node
+process.env.TS_NODE_FILES = true;
+require('ts-node').register({
+  dir: __dirname,
+});
+require('./index.ts');

--- a/index.sh
+++ b/index.sh
@@ -1,6 +1,0 @@
-#!/usr/bin/env bash
-
-# next line just for debug
-# set -x
-
-./node_modules/.bin/ts-node index.ts $@

--- a/index.ts
+++ b/index.ts
@@ -1,10 +1,10 @@
 const args = require('gar')(process.argv.slice(2));
-const package = require('./package.json');
+const pkg = require('./package.json');
 const fs = require('fs');
 const { organizeCode } = require('./vs-code-typescript-class-organizer/extension');
 
 if (args.h || args.help) {
-  console.log(`${package.name}#${package.version}
+  console.log(`${pkg.name}#${pkg.version}
     You can use next flags:
       [fileName]  Name of file to process. Required param.
       --useRegions   When true regions surounding member groups are added. True by default

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cli-typescript-class-organizer",
-  "version": "0.0.5",
+  "version": "0.0.6",
   "description": "This project should become a cli-version of amazing [TypeScript Class Organizer for VS Code](https://github.com/aljazsim/vs-code-typescript-class-organizer). So thanks to @aljazsim",
   "main": "index.js",
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -23,6 +23,6 @@
     "ts-node": "^8.0.3"
   },
   "bin": {
-    "tsco": "./index.sh"
+    "tsco": "bin.js"
   }
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cli-typescript-class-organizer",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "description": "This project should become a cli-version of amazing [TypeScript Class Organizer for VS Code](https://github.com/aljazsim/vs-code-typescript-class-organizer). So thanks to @aljazsim",
   "main": "index.js",
   "scripts": {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cli-typescript-class-organizer",
-  "version": "0.0.4",
+  "version": "0.0.5",
   "description": "This project should become a cli-version of amazing [TypeScript Class Organizer for VS Code](https://github.com/aljazsim/vs-code-typescript-class-organizer). So thanks to @aljazsim",
   "main": "index.js",
   "scripts": {

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -1,0 +1,5 @@
+{
+  "compilerOptions": {
+      "isolatedModules": false
+  }
+}

